tools/image_processing/imagej2/imagej2_analyze_skeleton/imagej2_analyze_skeleton.xml

```diff
@@ -78,18 +78,28 @@
         </test>
     </tests>
     <help>
+
+.. class:: warningmark
+
+The underlying AnalyzeSkeleton Fiji plugin works on binary images, so other image types will
+automatically be converted to binary before they are analyzed.  This step is performed using the
+ImageJ2 **Make Binary** command with the following settings: **Iterations:** 1, **Count:** 1,
+**Black background:** No, **Pad edges when eroding:** No.  If these settings are not appropriate,
+first manually convert the image to binary using the **Convert to binary (black and white) with
```

Comment:
@blankenberg @davebx BTW: there are 3 primary goals in setting up this ChIP-exo pipeline.  

1) Create a docker image that provides the Galaxy ChIP-exo flavor
2) Create a set of Galaxy workflows that provide the Galaxy version of the current Next-seq BioPipeline (the ChIP-exo tools)
3) Allow the above workflows to be run on Galaxy main

So goal 3 implies that all of the ChIP-exo tools are available on Galaxy main.  All tools in this pipeline are available on Galaxy main except these: https://github.com/galaxyproject/tools-iuc/issues/404

So it would be great if the tool migration could be scheduled such that these goals can be met in the short term. ;)